client/landing/gutenboarding/onboarding-block/animated-placeholder/index.tsx

```diff
@@ -9,7 +9,12 @@ import classnames from 'classnames';
  */
 import './style.scss';
 
-const AnimatedPlaceholder: FunctionComponent = ( { isSlow, texts } ) => {
+interface Props {
+	isSlow?: boolean;
```

Comment:
This is really `undefined` or `true`, isn't it? Setting this to `false` (the default) isn't something we want to do explicitly.

```suggestion
	isSlow?: true;
```

Reply:
Thanks!